src/Backend/Core/Js/utils.js

```diff
@@ -69,12 +69,13 @@ utils.cookies = {
     return null
   },
 
-  setCookie: function (name, value, days) {
+  setCookie: function (name, value, days, secure) {
     if (typeof days === 'undefined') days = 7
+    if (typeof secure === 'undefined') secure = location.protocol === 'https:'
 
     var expireDate = new Date()
     expireDate.setDate(expireDate.getDate() + days)
-    document.cookie = name + '=' + escape(value) + ';expires=' + expireDate.toUTCString() + ';path=/'
+    document.cookie = name + '=' + escape(value) + ';expires=' + expireDate.toUTCString() + ';path=/;secure=' + secure
```

Comment:
On [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Document/cookie#Security) and other examples I never see a boolean value being used, e.g. `;secure=true`. Is that supported?

If I run this snippet in my browser console:

```
document.cookie = "user=John; secure=false"
```

then the secure flag is toggled anyways? 
![image](https://user-images.githubusercontent.com/1352979/65502436-198a6500-dec3-11e9-8186-40100005197c.png)